.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+The template below is mostly useful for bug reports and support questions.
+Feel free to remove anything which doesn't apply to you and add more information where it makes sense.
+
+# Required information
+
+ * Distribution:
+ * Distribution version:
+ * Kernel version:
+ * LXC version:
+ * LXD version:
+ * Storage backend in use:
```

Comment:
Yeah, the output of "lxc info" would do, so long as it works. I'll try to update the markdown now.